docs/source/index.rst

```diff
@@ -41,7 +41,7 @@ This is the documentation for Mantid |release|.
    :width: 200px
    :align: right
 
-* :ref:`Algorithms List`
+* `Algorithms List <algorithm%20index/Algorithms/Algorithms.html>`_
```

Comment:
This should probably just say "Algorithms" now that it does not link to the full list.